core/src/mindustry/world/Tiles.java

```diff
@@ -27,64 +49,127 @@ public void each(Intc2 cons){
         }
     }
 
-    /** fills this tile set with empty air tiles. */
+    /**
+     * Fills this tile set with empty air tiles.
```

Comment:
lots of wasted space, extra '@'

Reply:
I honestly don't even know how that got there